controllers/progressivesync_controller.go

```diff
@@ -369,6 +369,13 @@ func (r *ProgressiveSyncReconciler) reconcileStage(ctx context.Context, ps syncv
 	if err != nil {
 		return syncv1alpha1.StageStatusFailed, err
 	}
+
+	// Skip stage if no clusters targeted
+	if len(selectedClusters.Items) == 0 {
+		log.Info("Skip stage as no clusters targeted", "stage", stage.Name)
```

Comment:
Just a little thing. Can you remove this logline here?

I'd like to file a new issue to change the `reconcileStage` function to return a `reason` so we can have a clear understanding of what's happening, but I'd prefer to avoid the single logline at this time.

Reply:
done